doc/source/changed_in_8.0.rst

```diff
@@ -0,0 +1,171 @@
+.. _changed_in_8.0:
+
+=============================
+Changed in Percona Server 8.0
+=============================
+
+|Percona Server| 8.0 is based on |MySQL| 8.0 and incorporates many of the
+improvements found in |Percona Server| 8.0.
+
+Features removed from |Percona Server| 8.0
+================================================================================
+
+Some features, that were present in |Percona Server| 5.7, are removed from
+|Percona Server| 8.0:
+
+Removed Features
+--------------------------------------------------------------------------------
+
+- Slow Query Log Rotation and Expiration
+- CSV engine mode for standard-compliant quote and comma parsing
+- Enforcing a storage engine
+- Utility user
+- Expanded program option modifiers
+- The ``ALL_O_DIRECT`` InnoDB flush method: it is not compatible with the new
+  redo logging implementation
+- XTRADB_RSEG table from INFORMATION_SCHEMA
+- InnoDB memory size information from SHOW ENGINE INNODB STATUS; the same
+  information is available from Performance Schema memory summary tables
+
+.. seealso::
+
+   |MySQL| Documentation: Performance Schema Table Description
```

Comment:
This only applies to the last item in the list above

Reply:
I would suggest to leave as it is: documentation is for human consumption: readers are able to figure out why that reference is there.